### PR TITLE
Update issue and PR templates

### PR DESCRIPTION
**Pull request by matthchr**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -7,9 +7,6 @@ Closes #[issue number]
 
 **Special notes for your reviewer**:
 
-**How does this PR make you feel**:
```
Comment (babbageclunk) [inline .github/pull_request_template.md:10]: I kind of liked this in the old template! Maybe not the giphy link.

Comment (davefellows) [reply .github/pull_request_template.md:10]: Yeah, I thought it was pretty cute but maybe it's a bit silly and unnecessary... I'm cool either way.

Comment (matthchr) [reply .github/pull_request_template.md:10]: I'm a curmudgeon and removed it but I can put it back if folks liked it =P

Comment (theunrepentantgeek) [reply .github/pull_request_template.md:10]: I kinda liked it, but not strongly motivated either way.

File: .github/ISSUE_TEMPLATE/improvement-template.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+labels: task
+assignees: ''
+
+---
+
+**Describe the current behavior**
+A description of how things are today.
+
+**Describe the improvement**
+How could things be changed or improved?
```
Comment (theunrepentantgeek) [inline .github/ISSUE_TEMPLATE/improvement-template.md:13]: Creating an issue is inherently opinionated, suggest making this _should_:

```suggestion
How should things be changed or improved?
```

**Pull request merged**